code_generator/Templates/UnityComponentConversionGenerator.tt

```diff
@@ -108,7 +108,9 @@ namespace <#= qualifiedNamespace #>
                 Unity.Entities.Entity entity;
                 if (!view.TryGetEntity(op.EntityId.Id, out entity))
                 {
-                    Debug.LogErrorFormat(TranslationErrors.OpReceivedButNoEntity, op.GetType().Name, op.EntityId.Id);
+                    view.LogDispatcher.HandleLog(new LogEvent("Entity not found during OnAddComponent in <#= componentDetails.ComponentName #>Translation.", LogType.Error)
```

Comment:
Simplify the message, since the information can be included via the attached data. (and again below)

Reply:
This should be `.Translation` everywhere.